torch/csrc/utils/python_arg_parser.h

```diff
@@ -47,14 +47,22 @@ struct FunctionParameter;
 struct FunctionSignature;
 struct PythonArgs;
 
+// Contains bounds Python arguments
```

Comment:
what is bounds?  Did you mean N or does this refer to something else?

Reply:
Should be "bound Python arguments" (as in argument binding)